src/main/scala/org/broadinstitute/clio/service/AuthorizationService.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.clio.service
+
+import akka.http.scaladsl.server.RequestContext
+import org.broadinstitute.clio.model._
+
+
+case class AuthorizationInfo(token: String, expiration: String, email: String, id: String)
+
+class AuthorizationService() {
+
+  def authorize(context: RequestContext): Boolean = {
+    val pairs = for (header <- context.request.headers) yield
+      header match {
+        case OidcAccessToken(token)      => 0 -> token
```

Comment:
It's not clear to me what the numbers here represent. Is it just a way to allow the code to allow the API to provide either `user id` or `sub` in the header? Is this a common idiom in Akka code?

IMO it might be clearer to have the code make a map using strings instead of numbers, e.g. `"token" -> token` and then you don't need `Vector` below to unpack them, you can pull them out directly using a map lookup.